person.rb

```diff
@@ -1,16 +1,27 @@
-class Person
-  attr_accessor :name, :age
+require_relative 'nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age, :rentals
   attr_reader :id
 
-  def initialize(age, parent_permission: true, name: 'Unknown')
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
```

Comment:
- Hey, let's also add the rentals in the constructor ` @rentals = []` to ensure we can push to the array as we are reading values from it.